Move to JSON require based system

thammi opened the pull request:
I wanted to use this package in my own project and implemented some improvements. I replaced the parsing of the files at runtime with require() on JSON files. This was needed to integrate the package in a browser based project and should perform better.

I also simplified the generation process. If you would like to merge the JSON require() part but not the adjustments to the algorithm I could split the commit up.

My version is currently published as readable-url-names.

thammi committed:
Move to JSON require based system

sharadbhat commented:
Hey man, that would be great! Although, could you split it up and keep only the imports and package.json updates?

Also, how is your generation process different from what I've implemented?

thammi commented:
I didn't really like the concept of "filler words" ("a", "the") and that they counted towards the word count. I removed them and simplified the generation a bit.

I will split it up, noticed that I should have done that the moment I pushed but was too lazy to fix immediately ;)
